### DISCUSSION TITLE
How to replace entire slot

### DISCUSSION THREAD

**Discussion by stevenbriscoeca** [microsoft/fluentui#29114] [Q&A]
Hello,

I'm trying to follow the doc here to replace an entire slot https://react.fluentui.dev/?path=/docs/concepts-developer-customizing-components-with-slots--page#replacing-the-entire-slot

I'm getting typescript errors :
![image](https://github.com/microsoft/fluentui/assets/3018441/95b21913-ac94-4ce9-9b11-40894c59b1ea)
![image](https://github.com/microsoft/fluentui/assets/3018441/8c678aec-b638-48f6-9159-8d0b384c33c7)

It doesn't seem like the doc has valid javascript either missing a "="

Thanks!

**Comment by layershifter** [comment]
@stevenbriscoeca it's a bug with typings, it's tracked in #27090.
